src/applications/gi/containers/VetTecEstimateYourBenefits.jsx

```diff
@@ -197,7 +200,12 @@ export class VetTecEstimateYourBenefits extends React.Component {
         </div>
         <div className="usa-width-one-half medium-6 columns vads-u-padding--1p5 medium-screen:vads-u-padding--0">
           <div className=" your-estimated-benefits">
-            <h3>Your estimated benefits</h3>
+            <h3
```

Comment:
This H3 will need a `tabIndex={-1}` to receive keyboard focus from our helper.

Reply:
It gets generated through our `platform/utils/ui `helper method, `focusElement()`